x-pack/platform/packages/shared/ai-infra/inference-common/src/chat_complete/api.ts

```diff
@@ -89,6 +89,11 @@ export type ChatCompleteOptions<
    * The list of messages for the current conversation
    */
   messages: Message[];
+  /**
+   * LLM temperature. All models support the 0-1 range (some supports more).
+   * Defaults to 0;
```

Comment:
Do all models currently default to `0`, or does this PR change their behavior by forcing the default to `0`?

Reply:
All models default to 0, we're not changing the default behavior here